statsmodels/stats/tests/test_weightstats.py

```diff
@@ -725,12 +731,13 @@ def test(self):
             assert_allclose(zstat, tc.statistic, rtol=1e-10)
             assert_allclose(pval, tc.p_value, rtol=1e-10, atol=1e-16)
 
-            zstat, pval = d1.ztest_mean(value=tc.null_value,
-                                       alternative=alternatives[tc.alternative])
+            zstat,
+            pval = d1.ztest_mean(value=tc.null_value,
```

Comment:
AFAIR = As Far As I Remember
https://en.wiktionary.org/wiki/AFAIR

Meaning: I'm pretty, but not completely sure, i.e. I would check an example if I had to write the code.